theme/templates/product.liquid

```diff
@@ -21,7 +21,7 @@
       <div class="px_625 pt2">
         {% render 'product-text-fields' product: product %}
       </div>
-      {% render 'add-to-cart-container' product: product %}
+      {% render 'add-to-cart-container' data_selector: 'data-mobile-product-add-to-cart-button' product: product %}
```

Comment:
I would personally move back to using the `isDesktop` flag, as it is more explicit to some reading this code. It also means, if there are other conditional data attributes, we won't need to pass them in